pkg/virt-handler/options_test.go

```diff
@@ -0,0 +1,171 @@
+package virthandler
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	"kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/api"
+)
+
+const memorySize = 2 * 1024 * 1024
+const memoryUnit = "KiB"
+const pageAmount = 1000
+
+var _ = Describe("Parsing VMI Options", func() {
+	Context("virt-handler VM processes VMI options during update", func() {
+		DescribeTable("it should convert api capabilities to cmdv1 topology", func(cellCount, pageCount, cpuCount int) {
+			caps := newLibvirtCapsTopology(
+				withCells(cellCount),
+				withCellMemory(memorySize, memoryUnit),
+				withCellPages(pageCount, pageAmount, memoryUnit),
+				withCellDistances(),
+				withCellCpus(cpuCount),
+			)
+			topology := caps.Host.Topology
+			expectedTopology := capabilitiesToTopology(caps)
```

Comment:
Please consider using a struct literal here instead of a function call - in order to make the output more explicit.